docs/markdown/developer_onboarding.md

```diff
@@ -26,6 +26,17 @@
 - **Automated tests.** `ninja test` or `ctest` exercises the bundled problem suite; for full GPU coverage, rely on the regression harness described earlier ([installation guide](installation.md), [quokka-tests.ini](https://github.com/quokka-astro/quokka/blob/development/regression/quokka-tests.ini#L65-L146)).
 - **Static analysis.** Run `clang-tidy` manually or via `scripts/tidy.sh` to match the repository’s CI checks, as documented in the How to Use clang-tidy guide ([clang-tidy how-to](howto_clang_tidy.md)).
 - **CUDA builds on macOS or Linux.** To build and test CUDA functionality locally, run `./scripts/bash/run-cuda-container.sh`. This script pulls the appropriate Docker image, launches a container, and performs a CUDA build—useful for catching CUDA-specific issues on your development machine.
+- **Writing and building documentation.** The developer is responsible for updating the documentation site. To build and view the documentation site locally:
+```bash
+cd docs
+pip install -r requirements.txt
+cd ..
+./build_docs.sh
```

Comment:
How about:
```
pip install -r docs/requirements.txt
./scripts/bash/build_docs.sh
./scripts/bash/view_docs.sh
```
and move `build_docs.sh` to `scripts/bash/` from its current location?